package.json

```diff
@@ -19,9 +19,10 @@
     "debug-server": "node tasks/serve-lib.js",
     "karma": "node tasks/test.js start test/karma.config.js",
     "transform-src": "jscodeshift --transform transforms/module.js src",
+    "changecase-src": "node tasks/filename-case-from-module.js",
     "transform-examples": "jscodeshift --transform transforms/module.js examples",
     "transform-test": "jscodeshift --transform transforms/module.js test",
-    "transform": "npm run transform-src && npm run transform-examples && npm run transform-test && npm run lint -- --fix"
+    "transform": "npm run transform-src && npm run changecase-src && npm run transform-examples && npm run transform-test && npm run lint -- --fix"
```

Comment:
I think we should be changing the case of file names *before* running the module transform on them.  In the end, we want `src/ol/Map.js` to look like:
```js
/**
 * @module ol/Map
 */
import _ol_View_ from './View.js';
// etc.
```

Reply:
Good point. Changing.

Reply:
Done, commit added.